test/test_jit_fuser.py

```diff
@@ -171,8 +168,9 @@ def f(x, y):
         x = torch.randn(2, 4, dtype=torch.float, device='cuda')
         y = torch.randn(1, 4, dtype=torch.float, device='cuda')
 
-        self.assertEqual(f(x, y).shape, (3, 4))
-        self.assertAllFused(f.graph_for(x, y))
+        scripted = self.checkScript(f, (x, y))
+        self.assertEqual(scripted(x, y).shape, (3, 4))
```

Comment:
nitpick: `self.assertEqual(scripted(x, y).shape, (3, 4))` might be unnecessary since `checkScript` compares whole tensors which includes their shapes as well.